Header header
string planner_id
geometry_msgs/PoseArray frontier_assignments
string[] robot_ids
string reasoning